ProjectSettings/ProjectSettings.asset

```diff
@@ -432,7 +432,8 @@ PlayerSettings:
   webGLUseEmbeddedResources: 0
   webGLUseWasm: 0
   webGLCompressionFormat: 1
-  scriptingDefineSymbols: {}
+  scriptingDefineSymbols:
+    14: 
```

Comment:
I haven't seen any changes to this file when building for UWP.

I was just curious if you had defined a new symbol.

It could also be because of your version of Unity being different from the repository.